internal/driver/mobile/app/GoNativeActivity.java

```diff
@@ -121,7 +121,7 @@ public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
 
                 // always place one character so all keyboards can send backspace
                 ignoreKey = true;
-                mTextEdit.setText("0");
+                mTextEdit.setText(" ");
```

Comment:
I don't know how this works but given that the issue notes a 0 being prefixed, would a space not show up as well? I'm thinking that setting empty string might make more sense?

Reply:
No, it must be 1 rune wide at least or the "backspace" will not be possible. This is required because what is in the Fyne Entry can be much more than the user has just typed into the current Android context.

My thinking is that suggestions on "0ar" doesn't offer sensible autosuggestion but " ar" will suggest "are" etc, based on even this peculiar keyboard using space characters as word separators.